db-migrations/000015_drop_legacy_schema.up.sql

```diff
@@ -0,0 +1,7 @@
+-- WARNING!!! this is not possible to roll back w/o back-fill of data from new schema or some other source
+-- no transaction as it would be supermassive
+DROP TABLE IF EXISTS aws_events_ips_hostnames; -- this automatically gets rid of partitions
```

Comment:
Question: I am bit concerned about our ability to not restore data, could we copy content from tables to other tables *_backup? And drop them after we are 100% sure

Reply:
We can keep one of the snapshots. Copying tables will take a ton of time. One more thing - we stopped dual-writes (i.e. writing to the tables we are dropping) couple months ago, so the data in old tables is very stale anyway.